python-main.js

```diff
@@ -524,6 +525,9 @@ function web_editor(config) {
         $('#editor').on('drop', doDrop);
         // Focus on the element with TAB-STATE=1
         $("#command-download").focus();
+
+        // A11y add aria-label to text area    
+        $('.ace_text-input').attr('aria-label', 'text editor')
```

Comment:
`text editor` should be a translatable string, and we need to add it to the en.js. es.js and pl.js

Reply:
Added in https://github.com/bbcmicrobit/PythonEditor/pull/215/commits/033671a81985f3e79f3bed954c61887fec8ed078
I pasted in Google translations for testing, you may want to check es.js for accuracy :)

Reply:
Spanish is good 👍 
This line still can be deleted now, no?

Reply:
Have  deleted this and tested that it still works